guides/common/modules/proc_upgrading-hosts-to-next-major-rhel-release.adoc

```diff
@@ -0,0 +1,42 @@
+ifndef::orcharhino[]
+** For {RHEL} 7 to {RHEL} 8 upgrade, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/planning-an-upgrade_upgrading-from-rhel-7-to-rhel-8[Planning an upgrade] in the _Upgrading from RHEL 7 to RHEL 8_ guide.
+** For {RHEL} 8 to {RHEL} 9 upgrade, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/upgrading_from_rhel_8_to_rhel_9/planning-an-upgrade_upgrading-from-rhel-8-to-rhel-9[Planning an upgrade] in the _Upgrading from RHEL 8 to RHEL 9_ guide.
+endif::[]
+* Prepare your {RHEL} hosts for the upgrade.
+ifndef::orcharhino[]
+** For {RHEL} 7 to {RHEL} 8 upgrade, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/upgrading_from_rhel_7_to_rhel_8/index#preparing-a-rhel-7-system-for-the-upgrade_upgrading-from-rhel-7-to-rhel-8[Preparing a RHEL 7 system for the upgrade] in the _Upgrading from RHEL 7 to RHEL 8_ guide.
+** For {RHEL} 8 to {RHEL} 9 upgrade, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/upgrading_from_rhel_8_to_rhel_9/assembly_preparing-for-the-upgrade_upgrading-from-rhel-8-to-rhel-9#preparing-a-rhel-8-system-for-the-upgrade_upgrading-from-rhel-8-to-rhel-9[Preparing a RHEL 8 system for the upgrade] in the _Upgrading from RHEL 8 to RHEL 9_ guide.
+endif::[]
+* Enable remote execution feature on {Project}.
+For more information, see xref:Configuring_and_Setting_Up_Remote_Jobs_{context}[].
+* Distribute {Project} SSH keys to the hosts that you want to upgrade.
+For more information, see xref:Distributing_SSH_Keys_for_Remote_Execution_{context}[].
+
+.Procedure
+. On {Project}, enable the Leapp plugin:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-installer} --enable-foreman-plugin-leapp
+----
+. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
+. Select the hosts that you want to upgrade to the next major {RHEL} version.
+. In the upper right of the Hosts window, from the *Select Action* list, select *Preupgrade check with Leapp*.
+. Click *Submit* to start the pre-upgrade check.
+. When the check is finished, click the *Leapp preupgrade report* tab to see if Leapp has found any issues on your hosts.
+Issues that have the *Inhibitor* flag are considered crucial and are likely to break the upgrade procedure.
+Some issues might have documentation linked that describe how to fix them.
+. Optional: If you have issues that have commands associated with them, you can fix them with a remote job.
+To do that, select these issues, click the *Fix Selected* button, and submit the job.
+. Once the issues are fixed, click the *Rerun* button, and then click *Submit* to run the pre-upgrade check again to verify that the {RHEL} hosts you are upgrading do not have any issues and are ready to be upgraded.
+. When your systems are ready for the upgrade, click the *Run Upgrade* button and click *Submit* to start the upgrade.
```

Comment:
```suggestion
. When your hosts are ready for the upgrade, click the *Run Upgrade* button and click *Submit* to start the upgrade.
```

How do I know when they're ready? Do you mean "After you have solved all issues on your hosts, ..."?